### PR TITLE
Finalize seeded canonical SenseNova workflow metadata

### PR DESCRIPTION
**Pull request by justinchuby**
## Summary

Follow-up to merged #540 that incorporates findings from real canonical-engine execution of the pinned SenseNova U1.5 package.

- derive deterministic image noise from generic `seed`/`width`/`height` request roles; caller latent remains an optional override
- keep positive and negative CFG prefixes and their frozen KV sequence symbols independent
- integrate the x0 flow from `t=0` to `t=1` using `v=(x0-z)/max(1-t,t_eps)`
- clamp generic pixel output to the declared `[-1, 1]` range
- add reusable image-noise geometry and tensor-clamp policy components
- regenerate the hashless canonical fixture and document production H200 evidence

No SenseNova/NEO dispatch or family-specific metadata fields are added.

## Validation

- `python3 -m pytest src/mobius/models/sensenova_u1_test.py src/mobius/generation/_policy_components_test.py -q --tb=short` — **78 passed**
- generated fixture exactly matches `tests/fixtures/onnx_genai_workflows/shared_state_pixel_flow/inference_metadata.yaml`
- targeted Ruff format/check and `git diff --check` pass

## Real canonical runtime evidence

Executed by ONNX GenAI `Engine::from_pipeline_dir` on H200 using the pinned 17.5B export (`1f6ec60423d29939dde4202fd82ae340b144e280`), not custom orchestration:

- text understanding prefill: logits `[1, 28, 151936]`
- 512×512 text-to-image, 20 steps: **6.99 s**
- 512×512 reference-image edit, 20 steps: **16.68 s**
- peak observed VRAM: **91,256 MiB**
- final outputs finite and bounded exactly to `[-1, 1]`

Published package: https://huggingface.co/justinchuby/sensenova-u1.5-8b-mot-onnx-canonical  
Upload commit: `f3f2eaf79f4991febee107cdabd0b27f0161b06e`

A clean 24 MB selective download of metadata, policies, graph descriptors, tokenizer, and processor assets passes canonical admission.

## Remaining runtime limitations

- text path currently validates understanding prefill, not full autoregressive decode
- generic Images/A1111 bindings cannot yet select a path-specific image chat template or construct reference-image placeholder masks for this unified package
- portable counter RNG is deterministic but intentionally does not reproduce PyTorch RNG, so final pixels are not directly comparable to #533's controlled-latent reference

Runtime support and synthetic executable coverage are in justinchuby/onnx-genai#1700.

### CODE REVIEW

**Comment by justinchuby** [comment]
The all-files `Lint` job reported format-only diagnostics in four files inherited from `main` (`gguf/_builder.py`, `gguf/_builder_test.py`, `gguf/_config_mapping_test.py`, and `rewrite_rules/_group_query_attention_test.py`). Commit b1bbab67 applies only the formatter output; the three associated test modules pass (119 tests). The SenseNova metadata/policy tests and changed-file `lintrunner` check also pass.

File: src/mobius/models/sensenova_u1_test.py [hunk 1]
```diff
@@ -522,7 +522,31 @@ def test_emits_complete_hashless_shared_state_workflow(self):
             "x0_velocity",
             "guidance_combine",
             "solver_step",
+            "image_noise_geometry",
+            "image_noise",
+            "latent_scale",
         }
```
[the remark this answers is not in the document]
Comment (justinchuby) [reply src/mobius/models/sensenova_u1_test.py:531]: Added `image_output_clamp` to the pinned canonical component set in 72f8b8fb and reran the targeted metadata test plus Ruff checks.

File: src/mobius/integrations/gguf/_builder.py [hunk 2]
```diff
@@ -812,35 +812,31 @@ def _reorder_deltanet_v_heads(state_dict: dict, config) -> dict:
...
         return base.reshape(-1)
 
     v_rows = _expand(head_perm, head_v_dim)  # length value_dim
 
-    def _index_dim0(t: "torch.Tensor", idx: "torch.Tensor") -> "torch.Tensor":
+    def _index_dim0(t: torch.Tensor, idx: torch.Tensor) -> torch.Tensor:
         return t.index_select(0, idx)
 
-    def _index_dim1(t: "torch.Tensor", idx: "torch.Tensor") -> "torch.Tensor":
+    def _index_dim1(t: torch.Tensor, idx: torch.Tensor) -> torch.Tensor:
```
[the remark this answers is not in the document]
Comment (justinchuby) [reply src/mobius/integrations/gguf/_builder.py]: Removed exactly the unused local `_index_dim1` helper in 9aa399d9; `_index_dim0` and all call sites are unchanged. Targeted GGUF tests pass (83), and full `lintrunner f --output oneline --all-files` completed successfully.

**Comment by justinchuby** [comment]
Final lint/review status:

- removed unused `_index_dim1` exactly as requested in `9aa399d9`
- targeted GGUF tests: **83 passed**
- full `lintrunner f --output oneline --all-files`: pass
- all review threads: resolved
- current CI: every completed check is green; only `Integration (fast)` remains queued

I also compared #544's four-file Ruff patch against this head. #545 contains every current-main Ruff correction from #544 in the same four files; relative to #544 it only removes the now-reported unused helper and uses equivalent D205-compliant docstring wording. Thus #545 functionally subsumes #544 despite no commit ancestry. After #545 merges, #544 can be closed as redundant.

**Comment by justinchuby** [comment]
Final rubber-duck CFG/geometry blocker fixed in `2d3f107a`:

- invokes `image_grid_positions` independently for positive and negative prompt tensors
- binds conditional/unconditional denoisers to their corresponding position IDs and token grids
- derives downstream resolution/noise-scale conditioning from the selected latent's actual dimensions; request dimensions only construct runtime-generated noise
- adds runtime policy coverage proving temporal position outputs differ for prompt lengths 5 vs 2 while spatial grids remain equal
- pins the exact branch-specific metadata wiring and latent-geometry dataflow

Validation:
- SenseNova + policy tests: **79 passed**
- workflow artifact binding tests: **14 passed**
- ONNX GenAI executable shared-state conformance: **2 passed**
- regenerated fixture comparison and SHA: pass (`9dd15af58703f0ef163abb17f7008e92f58b3c7a4071784b1b8e53c05b411abd`)
- full lintrunner: pass

**Comment by justinchuby** [comment]
Executable regression upgraded in `bfb7d2dc`:

- generation embedding now emits one token per actual latent grid cell and consumes resolved `noise_scale`
- denoiser prediction numerically includes branch temporal `position_ids`, `token_grid`, and independent conditional/unconditional cache effects
- denoiser output shape is derived from the actual token grid rather than fixed request geometry

The paired ONNX GenAI conformance assertion is in justinchuby/onnx-genai#1700 at `1810aa30e`: positive/negative lengths 5/2, supplied latent 64×96, request 32×32, exact final pixel assertion, plus explicit rejection of shared-position and request-scale counterfactual outputs.

Validation: generator + fixture comparison pass; executable conformance 2/2; SenseNova/workflow tests 39 passed; full lintrunner passed.
